fragment.go

```diff
@@ -803,29 +873,37 @@ func (f *fragment) min(filter *Row, bitDepth uint) (min, count uint64, err error
 		return 0, 0, nil
 	}
 
-	for i := bitDepth; i > uint(0); i-- {
-		ii := i - 1 // allow for uint range: (bitDepth-1) to 0
-		row := f.row(uint64(ii))
+	// Find highest negative number w/o sign and negate, if negatives are available.
```

Comment:
I found this comment confusing—aren't we finding the highest number *with* the sign bit set?

Reply:
It's the highest negative number, which is to say, the one closest to being positive.

Reply:
er, wait.

it's the negative number which is highest, if you ignore its sign. okay, conceded, the comment is probably confusing.